src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private static FruitDao fruitDao;
+
+    @BeforeClass
+    public static void beforeClass() {
+        Storage.fruitStorage.clear();
+        fruitDao = new FruitDaoImpl();
+    }
+
+    @Before
+    public void setUp() {
+        Storage.fruitStorage.put("banana", 80);
+    }
+
+    @Test
+    public void addFruit_AddToDB_Ok() {
+        fruitDao.addFruit("orange", 20);
+        int actual = Storage.fruitStorage.size();
+        Assert.assertEquals(2, actual);
+    }
+
+    @Test
+    public void addFruit_AddToDB_NotOk() {
+        fruitDao.addFruit("kiwi", 200);
+        fruitDao.addFruit("lemon", 5);
+        int actual = Storage.fruitStorage.size();
+        Assert.assertFalse(Storage.fruitStorage.isEmpty());
+        Assert.assertNotEquals(2, 3);
+    }
+
+    @Test
+    public void getQuantity_GetFruitQuantity_Ok() {
+        int actual = fruitDao.getQuantity("banana");
+        Assert.assertEquals(80, actual);
+    }
+
+    @Test
+    public void getQuantity_GetFruitQuantity_NotOk() {
+        int actual = fruitDao.getQuantity("banana");
+        Assert.assertNotEquals(20, 80);
+    }
```

Comment:
It is bad test, because it didn't check anything for real, you just compare two variables - 20 and 80